aretomo/__init__.py

```diff
@@ -68,6 +74,8 @@ def getEnviron(cls):
         cudaLib = cls.getVar(ARETOMO_CUDA_LIB, pwem.Config.CUDA_LIB)
         environ.addLibrary(cudaLib)
 
+        environ.set("LD_LIBRARY_PATH","")
```

Comment:
???

Reply:
upss! I was testing this. Let me remove it but this was testing a case where LD_LIBRARY_PATH was polluted with many cudas paths

Thanks for looking at this. My tests worked.